docs/create_dimensions.md

```diff
@@ -13,12 +13,35 @@ In order to successfully push your JSON schema into the Monitor platform, you wi
 1. In the hamburger menu on the left side of the screen of Maximo Asset Monitor, click `Connect`.
```

Comment:
General comment is we need to weave this into telling the broader story of why dimensions are important and that is missing right now.   The story line of why you are adding these dimension values.   Dimension values can be use for aggregation or for picking a specific entity in a summary dashboard.